codechecker_lib/database_handler.py

```diff
@@ -457,6 +457,9 @@ def __init__(self, workspace, migration_root, run_env=None):
         self.dbpath = os.path.join(workspace, 'codechecker.sqlite')
         self.run_env = run_env
     
+        if not os.path.exists(workspace):
```

Comment:
I think creating the workspace should be done in the arg_handler.py [handle_check](https://github.com/Ericsson/codechecker/blob/master/codechecker_lib/arg_handler.py#L202).
Why did this [L212](https://github.com/Ericsson/codechecker/blob/master/codechecker_lib/arg_handler.py#L212) in handle_check failed to create the workspace for sqlite?